<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Video</string>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QLabel" name="labelFrameRate" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="labeImageSize" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Image size :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Frame rate :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string># of frames : </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Codec 4CC :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Video duration :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Aspect ratio :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QLabel" name="LabelAspectRatio" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLabel" name="labelVideoDuration" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="labelNbOfFrames" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLabel" name="label4CC" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Extra Properties</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Quarter Pixel :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="LabelQuarterPixel" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>GMC :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="LabelPackedBitstream" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Packed Bitstream :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="LabelGMC" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Audio</string>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="4" column="1" >
       <widget class="QLabel" name="labelAudioDuration" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_15" >
        <property name="text" >
         <string>Bitrate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="labelBitrate" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLabel" name="labelFrequency" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_14" >
        <property name="text" >
         <string>VBR</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_13" >
        <property name="text" >
         <string>Audio duration :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QLabel" name="labelVBR" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="labelChannels" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="labelACodec" >
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Channels</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_12" >
        <property name="text" >
         <string>Codec</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Frequency</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
